### PR TITLE
[fix bug 1302415] Use more descriptive filenames in admin exports.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: mozillians/common/mixins.py [hunk 1]
```diff
@@ -42,6 +44,17 @@ def async_data_export(file_format, values_list, qs_model, filename):
 
 
 class S3ExportMixin(ExportMixin):
+
+    def get_export_filename(self, file_format):
+        query_string = self.request.GET.urlencode().replace('=', '_')
+        if query_string == '':
+            query_string = 'All'
+        date_str = datetime.now().strftime('%Y-%m-%d-%H-%m-%s')
```
Comment (johngian) [inline mozillians/common/mixins.py]: Please change that format to:
```
date_str = datetime.now().strftime('%Y-%m-%d-%H:%m:%s')
```

File: mozillians/common/mixins.py [hunk 2]
```diff
@@ -42,6 +44,17 @@ def async_data_export(file_format, values_list, qs_model, filename):
 
 
 class S3ExportMixin(ExportMixin):
+
+    def get_export_filename(self, file_format):
+        query_string = self.request.GET.urlencode().replace('=', '_')
```
Comment (johngian) [inline mozillians/common/mixins.py]: nit: For consistency you can replace ``query_string`` with ``query_str``

**Comment by johngian** [comment]
r+wc 👍

**Pull request merged**